dll/win32/mshtml/olecmd.c

```diff
@@ -241,6 +241,13 @@ static HRESULT exec_print(HTMLDocument *This, DWORD nCmdexecopt, VARIANT *pvaIn,
         return S_OK;
     }
 
+#ifdef __REACTOS__
+    // returning here fixes CORE-16884. Maybe use this until printing works.
+    ERR("Sorry, ReactOS does not support printing yet.\n");
+    return S_OK;
```

Comment:
No cleanup?

Reply:
Please note that just above this code in Line #241 we have a similar return that does not do any cleanup, so it seems that none should be needed here either.

Reply:
Line 241 is the failure case, proposed code is in the success case...

Reply:
241 has no cleanup because retrieving the object that should be cleaned up failed.

Reply:
Please insert this between Lines #246 (ERR) and Line #247 (return).
```    nsIWebBrowserPrint_Release(nsprint);```
Thanks @learn-more for the hints.